spikingjelly/activation_based/triton_kernel/flex_sn_inductor/__init__.py

```
@@ -0,0 +1,9 @@
+try:
+    from .hop import flex_sn_scan, FlexSNScan, eager_scan
+except BaseException as e:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `BaseException`

(BLE001)